dashboard/lib/services/script_seed.rb

```diff
@@ -421,6 +425,31 @@ def self.import_lessons_resources(lessons_resources_data, seed_context)
       LessonsResource.joins(:lesson).where('stages.script_id' => seed_context.script.id)
     end
 
+    def self.import_scripts_resources(scripts_resources_data, seed_context)
+      return [] unless seed_context.script.get_course_version
+      return [] unless scripts_resources_data
+
+      scripts_resources_to_import = scripts_resources_data.map do |lr_data|
+        resource_id = seed_context.resources.select {|r| r.key == lr_data['seeding_key']['resource.key']}.first&.id
+        raise 'No resource found' if resource_id.nil?
+
+        ScriptsResource.new(
+          script_id: seed_context.script.id,
+          resource_id: resource_id
+        )
+      end
+
+      # destroy_outdated_objects won't work on ScriptsResource objects because
+      # they do not have an id field. Work around this by inefficiently deleting
+      # all ScriptsResources using 1 query per lesson, and then re-importing all
+      # ScriptsResources in a single query. It may be possible to eliminate
+      # these extra queries by adding an id column to the ScriptsResource model.
+      seed_context.script.resources = []
```

Comment:
the code looks right, but the comment looks like it needs to be updated.

Reply:
ah, I'd updated most of it but missed a part. Nice catch!